build/lib/pgBackRestBuild/Config/Data.pm

```diff
@@ -0,0 +1,2234 @@
+        &CFGBLDOPT_RULE_COMMAND =>
+        {
+            &CFGBLDCMD_ARCHIVE_GET => {},
+            &CFGBLDCMD_ARCHIVE_PUSH => {},
+            &CFGBLDCMD_BACKUP => {},
+            &CFGBLDCMD_CHECK => {},
+            &CFGBLDCMD_EXPIRE => {},
+            &CFGBLDCMD_INFO => {},
+            &CFGBLDCMD_LOCAL => {},
+            &CFGBLDCMD_REMOTE => {},
+            &CFGBLDCMD_RESTORE => {},
+            &CFGBLDCMD_STANZA_CREATE => {},
+            &CFGBLDCMD_STANZA_UPGRADE => {},
+        }
+    },
+
+    &CFGBLDOPT_COMPRESS_LEVEL_NETWORK =>
+    {
+        &CFGBLDOPT_RULE_SECTION => CONFIG_SECTION_GLOBAL,
+        &CFGBLDOPT_RULE_TYPE => CFGBLDOPT_TYPE_INTEGER,
+        &CFGBLDOPT_RULE_DEFAULT => CFGBLDOPTDEF_DEFAULT_COMPRESS_LEVEL_NETWORK,
+        &CFGBLDOPT_RULE_ALLOW_RANGE => [CFGBLDOPTDEF_DEFAULT_COMPRESS_LEVEL_NETWORK_MIN, CFGBLDOPTDEF_DEFAULT_COMPRESS_LEVEL_NETWORK_MAX],
+        &CFGBLDOPT_RULE_COMMAND =>
+        {
+            &CFGBLDCMD_ARCHIVE_GET => {},
+            &CFGBLDCMD_ARCHIVE_PUSH => {},
+            &CFGBLDCMD_BACKUP => {},
+            &CFGBLDCMD_CHECK => {},
+            &CFGBLDCMD_EXPIRE => {},
+            &CFGBLDCMD_INFO => {},
+            &CFGBLDCMD_LOCAL => {},
+            &CFGBLDCMD_REMOTE => {},
+            &CFGBLDCMD_RESTORE => {},
+            &CFGBLDCMD_STANZA_CREATE => {},
+            &CFGBLDCMD_STANZA_UPGRADE => {},
+        }
+    },
+
+    &CFGBLDOPT_NEUTRAL_UMASK =>
+    {
+        &CFGBLDOPT_RULE_SECTION => CONFIG_SECTION_GLOBAL,
+        &CFGBLDOPT_RULE_TYPE => CFGBLDOPT_TYPE_BOOLEAN,
+        &CFGBLDOPT_RULE_DEFAULT => CFGBLDOPTDEF_DEFAULT_NEUTRAL_UMASK,
+        &CFGBLDOPT_RULE_COMMAND =>
+        {
+            &CFGBLDCMD_ARCHIVE_GET => {},
+            &CFGBLDCMD_ARCHIVE_PUSH => {},
+            &CFGBLDCMD_BACKUP => {},
+            &CFGBLDCMD_CHECK => {},
+            &CFGBLDCMD_EXPIRE => {},
+            &CFGBLDCMD_LOCAL => {},
+            &CFGBLDCMD_REMOTE => {},
+            &CFGBLDCMD_RESTORE => {},
+            &CFGBLDCMD_STANZA_CREATE => {},
+            &CFGBLDCMD_STANZA_UPGRADE => {},
+            &CFGBLDCMD_START => {},
+            &CFGBLDCMD_STOP => {},
+        }
+    },
+
+    &CFGBLDOPT_CMD_SSH =>
+    {
+        &CFGBLDOPT_RULE_SECTION => CONFIG_SECTION_GLOBAL,
+        &CFGBLDOPT_RULE_TYPE => CFGBLDOPT_TYPE_STRING,
+        &CFGBLDOPT_RULE_DEFAULT => CFGBLDOPTDEF_DEFAULT_CMD_SSH,
+        &CFGBLDOPT_RULE_COMMAND =>
+        {
+            &CFGBLDCMD_ARCHIVE_GET => {},
+            &CFGBLDCMD_ARCHIVE_PUSH => {},
+            &CFGBLDCMD_BACKUP => {},
+            &CFGBLDCMD_CHECK => {},
+            &CFGBLDCMD_INFO => {},
+            &CFGBLDCMD_LOCAL => {},
+            &CFGBLDCMD_RESTORE => {},
+            &CFGBLDCMD_STANZA_CREATE => {},
+            &CFGBLDCMD_STANZA_UPGRADE => {},
+            &CFGBLDCMD_START => {},
+            &CFGBLDCMD_STOP => {},
+            &CFGBLDCMD_EXPIRE => {},
```

Comment:
If we're going with "I making all these changes anyway" then can we please put cmd_expire in alphabetical order here?